app/src/main/java/com/cornellappdev/android/eateryblue/ui/components/home/PaymentWidgets.kt

```diff
@@ -0,0 +1,59 @@
+package com.cornellappdev.android.eateryblue.ui.components.home
```

Comment:
We probably want this to be in a new package directory since this isn't really pertaining to `home`. Feel free to make a new `details` or similar package to put these components in.